exercises/04_power/power.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import "fmt"
+
+func Pow(x, y int) int {
+	if y == 0 {
+		x = 1
+		return x
+	}
+	if x == 0 {
+		x = 0
+		return x
+	}
+	for tmp := x; y != 1; y-- {
+		x = x * tmp
+
+	}
+	return x
+}
+
+func main() {
+	fmt.Println("Program determines x to the power of y")
+	fmt.Println("Enter x and y:")
+	var x, y int
+	fmt.Scanln(&x, &y)
+	if y < 0 {
```

Comment:
That's great that you've added this check to your main, but it isn't tested by your program. Please test negative `y`